components/common/src/command/package/install.rs

```diff
@@ -164,6 +165,12 @@ impl FromStr for InstallSource {
     }
 }
 
+impl std::convert::TryFrom<&str> for InstallSource {
+    type Error = habitat_core::Error;
+
+    fn try_from(s: &str) -> StdResult<Self, Self::Error> { InstallSource::from_str(s) }
```

Comment:
Similarly here... is there benefit of having a `TryFrom` if we already have `FromStr`?